openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -1,31 +1,62 @@
-openapi: 3.1.0
+openapi: 3.0.1
 info:
   title: p4pa-workflow-hub
   description: Api and Models
-  version: "1.0.0"
+  version: 0.0.1
+servers:
+  - url: "http://localhost:8080/workflowhub"
 paths:
-  /api/v1/greet:
+  /{workflowId}/status:
```

i would set a prefix, instead of having an id directly after the root
```suggestion
  /workflows/{workflowId}/status:
```

---

I thought about the prefix here, i didn't put it because we already have the name of the app in the prefix, so i preferred to have the url cleaner, in this case we wouldn't have `http://localhost:8080/workflowhub/workflows/{workflowId}/status`
But works the same ;-)

---

I get your point, actually this application will not handle any other entity, just workflows, but let's keep the path as requested ;)